src/test/java/uk/gov/companieshouse/overseasentitiesapi/controller/CostsControllerTest.java

```diff
@@ -2,31 +2,51 @@
 
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 import uk.gov.companieshouse.api.model.transaction.Transaction;
+import uk.gov.companieshouse.overseasentitiesapi.exception.ServiceException;
 import uk.gov.companieshouse.overseasentitiesapi.service.CostsService;
+import uk.gov.companieshouse.overseasentitiesapi.service.OverseasEntitiesService;
+import uk.gov.companieshouse.overseasentitiesapi.service.SubmissionType;
 
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
 
 @ExtendWith(MockitoExtension.class)
 class CostsControllerTest {
+
+    private final static String OVERSEAS_ENTITY_ID = "testId";
+
     private final static String REQUEST_ID = "7567675jgjhgh";
 
     private final Transaction transaction = new Transaction();
 
     @Mock
-    private CostsService costService;
+    private OverseasEntitiesService overseasEntitiesService;
+
+    @Test
+    void getRegistrationCosts() throws ServiceException {
```

Comment:
cost singular?

Reply:
Does the team have any naming standards for tests? If not we should have a chat about it as we should be naming tests like `testGetCostReturnsRegisterCost`. I think other tests in the repo are named similarly.